TOC.json

```diff
@@ -1,77 +1,141 @@
 {
+    "defaultSpecLocations": {
+        "apiSpecPath": "../API_Specs",
+        "apiSpecGitUrl": "https://raw.githubusercontent.com/PlayFab/API_Specs/master",
+        "apiSpecPfUrl": "https://www.playfabapi.com/apispec"
+    },
+    "docFieldDescriptions": {
+        "name": "Full descriptive name used for describing this document to GUI",
+        "shortname": "Short descriptive name used for describing this document to GUI",
+        "shortname2": "Will replace shortname after transition phase",
```

Comment:
May i suggest to give the new properties names that we wouldn't have to change again after transition phase? For example, "shortName" instead of "shortname2" and 'relPath" instead of "path2". I'd rather keep those than having "shortname2" and "path2" when we remove "shortname" and "path"

Reply:
Yeah, I was having difficulty coming up with names that I wanted to use, which accurately described the new fields.
And you solved it in seconds.  👍